### DISCUSSION TITLE
Will RxSwift 6.0 eventually be integrated in this repository?

### DISCUSSION THREAD

**Discussion by c137Choi** [Polidea/RxBluetoothKit#399] [General]
Though I am using a private repository with latest RxSwift integrated, I prefer an offical version of this.😀

**Comment by chadbag** [comment]
I've been put on a project that uses Cocoapods and has a line for RxBluetoothKit and specifies a 7.0.2 tag.  To get this it had to point specifically to a different project (fork) on GitHub that is unfortunately no longer available. However if we just specify pod RxBluetoothKit we get 7.0.0.    But it appears from doing searches with the search engines that 6.0.0 is the latest "official" release.  Can someone help me understand the state of this?  This appears to be the "official" release as the various package managers point back here and I am not really sure as I am just being asked to work on this and didn't create this part of our project myself.
